azurerm/internal/services/storage/storage_account_network_rules_resource.go

```diff
@@ -93,11 +93,33 @@ func resourceStorageAccountNetworkRules() *schema.Resource {
 					string(storage.DefaultActionDeny),
 				}, false),
 			},
+
+			"private_endpoint_access_rules": {
```

Comment:
endpoints are part of private link? so
```suggestion
			"private_link_access": {
```

as these are not rules? so much as granting access to these private links